docs/source/installation.md

```diff
@@ -85,3 +112,10 @@ To perform docking you must provide `boxcenter` (`center` for AutoDock-Vina) and
 - Visit the [moldrug's Docker container](https://hub.docker.com/r/ale94mleon/4moldrug).
 
 Finally, `pip install moldrug` inside the container.
+
+## Running tests
+
+```sh
+pip install -e .[dev]
+pip install pytest
```

Comment:
Not needed as [dev] already includes pytest. Maybe also include the command to actually run the tests `pytest tests`